apps/studio/components/interfaces/Integrations/Wrappers/Wrappers.constants.ts

```diff
@@ -3316,4 +3317,140 @@ export const WRAPPERS: WrapperMeta[] = [
       },
     ],
   },
+  {
+    name: 'cfd1_wrapper',
+    description: 'Read and write data from Cloudflare D1 databases using the Wasm FDW.',
+    handlerName: WRAPPER_HANDLERS.CLOUDFLARE_D1,
+    validatorName: 'wasm_fdw_validator',
+    icon: `${BASE_PATH}/img/icons/cloudflare-icon.svg`,
+    extensionName: 'Cfd1Fdw',
+    label: 'Cloudflare D1',
+    docsUrl: 'https://fdw.dev/catalog/cfd1/',
+    minimumExtensionVersion: '0.1.0',
+    server: {
+      options: [
+        {
+          name: 'fdw_package_url',
+          label: 'FDW Package URL',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+          defaultValue:
+            'https://github.com/supabase/wrappers/releases/download/wasm_cfd1_fdw_v0.1.0/cfd1_fdw.wasm',
+          hidden: true,
+        },
+        {
+          name: 'fdw_package_name',
+          label: 'FDW Package Name',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+          defaultValue: 'supabase:cfd1-fdw',
+          hidden: true,
+        },
+        {
+          name: 'fdw_package_version',
+          label: 'FDW Package Version',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+          defaultValue: '0.1.0',
+          hidden: true,
+        },
+        {
+          name: 'fdw_package_checksum',
+          label: 'FDW Package Checksum',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+          defaultValue: '783232834bb29dbd3ee6b09618c16f8a847286e63d05c54397d56c3e703fad31',
+          hidden: true,
+        },
+        {
+          name: 'api_url',
+          label: 'API URL',
+          required: false,
+          encrypted: false,
+          secureEntry: false,
+          defaultValue: 'https://api.cloudflare.com/client/v4/accounts/<account_id>/d1/database',
+        },
+        {
+          name: 'account_id',
+          label: 'Account ID',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+        },
+        {
+          name: 'database_id',
+          label: 'Database ID',
+          required: true,
+          encrypted: false,
+          secureEntry: false,
+        },
+        {
+          name: 'api_token_id',
+          label: 'Cloudflare D1 API Token',
+          required: true,
+          encrypted: true,
+          secureEntry: true,
+          urlHelper: 'https://dash.cloudflare.com/profile/api-tokens',
+        },
+      ],
+    },
+    tables: [
+      {
+        label: 'Databases',
```

Comment:
nit - just wondering if this should also say "D1 Databases"

Reply:
![image](https://github.com/user-attachments/assets/598e4d5d-3521-41bb-aa50-74967e674321)